### PR TITLE
feat: home hero scroll

### LINKED ISSUE #38: Hero: scroll button
Screenshot 2023-10-12 at 17 17 42

### PR DESCRIPTION
**Pull request by dicagno**
Hero scroll btn enabled on the **home** page.

Fix #38 

Test URLs:
- Before: https://main--bevespi--hlxsites.hlx.page/
- After: https://feat-hero-scroll-btn--bevespi--hlxsites.hlx.page/

### CODE REVIEW

File: blocks/hero/hero.css [hunk 1]
```diff
@@ -3,22 +3,66 @@
...
+
 main .hero {
   position: relative;
   padding: 30px 0;
   min-height: 71px;
 }
 
 main .hero.home {
+  height: 100%;
+  border: 1px solid white;
```
Comment (paolomoz) [inline blocks/hero/hero.css:19]: why this border?

File: blocks/hero/hero.css [hunk 2]
```diff
@@ -154,4 +192,18 @@ main .hero > div:nth-of-type(n+2) div:nth-of-type(2) p:nth-of-type(n+2) > pictur
...
+    background-size: 44px 20px, cover;
+  }
+
+  main .hero.home .btn-scroll img {
+    width: 32px;
+    height: 32px;
+  }
+
+  main .hero.home .btn-scroll a {
+    font-size: 1.6rem;
```
Comment (paolomoz) [inline blocks/hero/hero.css:207]: would it make sense to use --body-font-size-m here or is this a special case?

**Review by paolomoz — APPROVED** [review]
@dicagno approved with 2 comments. the comment about the border is more important :)

File: blocks/hero/hero.css (hunk 1 above)
Comment (dicagno) [reply blocks/hero/hero.css:19]: leftover, thanks paolo

File: blocks/hero/hero.css (hunk 2 above)
Comment (dicagno) [reply blocks/hero/hero.css:207]: You mean `--heading-font-size-s` right?

Comment (dicagno) [reply blocks/hero/hero.css:207]: good point!

**Pull request merged**